src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -1936,6 +1950,11 @@ public void ReplaceIndexes(string oldIndexName, string replacementIndexName, Can
                             oldIndexDefinition.LockMode.HasValue)
                             newIndex.SetLock(oldIndexDefinition.LockMode.Value);
 
+                        if (newIndex.Definition.ArchivedDataProcessingBehavior == ArchivedDataProcessingBehavior.ExcludeArchived &&
+                            newIndexDefinition.ArchivedDataProcessingBehavior.HasValue == false &&
+                            oldIndexDefinition.ArchivedDataProcessingBehavior.HasValue)
+                            newIndex.Definition.ArchivedDataProcessingBehavior = oldIndexDefinition.ArchivedDataProcessingBehavior.Value;
```

Comment:
Why do we have this logic? What is it supposed to do?

Reply:
removed, gonna check if the UI won't break